api/rooms/tests/test_detail.py

```diff
@@ -1 +1,106 @@
-from django.test import TestCase
\ No newline at end of file
+from django.test            import TestCase,Client
+
+from api.rooms.models import Location, Option, RoomType, TradeType, Room, OptionType, RoomImage, RoomOption
+
+class RoomViewTest(TestCase):
+    def setUp(self):
+
+        Location.objects.create(id='1',state='서울특별시',city='강남구',dong='삼성동',
+                    detail='테헤란로 427',latitude='37.50637292',longitude='127.0536407',
+                    utmk_x='1945020.83739406',utmk_y='960738.939126282')
+
+        RoomType.objects.bulk_create([
+            RoomType(id='1',name='ONE_ROOM'),
+            RoomType(id='2',name='MULTI_ROOM'),
+            RoomType(id='3',name='OFFICETEL')
+        ])
+
+        TradeType.objects.bulk_create([
+            TradeType(id='1',name='MONTHLY_RENT'),
+            TradeType(id='2',name='DEPOSIT'),
+            TradeType(id='3',name='SALE')
+        ])
+
+        a = Room.objects.create(id='1',title='원룸1',content='매물정보 원룸 1 풀옵션 ',
+                        location_id='1',room_type_id='1',trade_type_id='1',monthly_rent='30',
+                        deposit='100',sale='0',agent='성규부동산',floor='1',room_area='10.12',
+                        room_count='1',bathroom_count='1',heating_type='개별',elevator='유')
+        a.created_at = '2021-06-28T19:50:19.403'
```

Comment:
생성할때, created_at을 추가할 수 없나요??

```suggestion
        a = Room.objects.create(id='1',title='원룸1',content='매물정보 원룸 1 풀옵션 ',
                        location_id='1',room_type_id='1',trade_type_id='1',monthly_rent='30',
                        deposit='100',sale='0',agent='성규부동산',floor='1',room_area='10.12',
                        room_count='1',bathroom_count='1',heating_type='개별',elevator='유', created_at = '2021-06-28T19:50:19.403')
```

Reply:
생성할때 추가하는 방식으로 해보았는데 ! `created_at = models.DateTimeField(auto_now_add=True)` 자동으로 생성 날짜가 생겨서 오류가 나는것 같아요 !